screwdriver.yaml

```diff
@@ -18,6 +18,7 @@ jobs:
             - coverage: ./ci/coverage.sh
```

Comment:
do we still need coverAll? some steps can be removed if we don't need it?

Reply:
let's keep it.